addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5330,7 +5339,7 @@ msgstr ""
 
 #: unknown
 msgctxt "#13109"
-msgid "Test & apply resolution"
+msgid "Test & apply the resolution"
```

Comment:
please note that in order to keep settings labels short we don't use perfect grammar on these. So "Test & apply resolution" is perfectly fine for a settings label. In settings descriptions we may use correct grammar, but within the UI and settings it should be a mixture between being correct but not too long and still not wrong.